webapp/src/main/java/io/github/microcks/util/asyncapi/AsyncAPIImporter.java

```diff
@@ -87,10 +87,14 @@ else if (line.startsWith("---") || line.startsWith("asyncapi: ")) {
          ObjectMapper mapper = null;
          if (isYaml) {
             mapper = new ObjectMapper(new YAMLFactory());
+            // Jackson YAML parser can;t deal with any quotes around "$ref" and double
+            // quotes around the path.
+            specContent = specContent.replaceAll("[\\\"']?\\$ref[\\\"']?:\\s*[\\\"'](#.*)[\\\"']", "\\$ref: '$1'")
+                  .replaceAll("[\\\"']?pattern[\\\"']?:\\s*[\\\"'](.*)[\\\"']", "pattern: $1");
```

Comment:
We should face the same issue in OpenAPIImporter too? Correct? Would you have any sample or unit tests that demonstrates that one so that we could replicate?

Reply:
I'll add a unit test for asyncapi and one for openapi and verify if the same fix is required there. I'll create a separate PR for this